app/controllers/admin/diplomas_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Admin::DiplomasController < ActionController::Base
+  respond_to :js
+  before_action :load_diploma, except: [:index, :new, :create]
+  before_action :verify_admin!
+
+  def index
+    @diplomas = Diploma.all
+    @diplomas_pending=Diploma.pending
```

Comment:
Trước và sau toán tử phải có dấu cách nhé:
`@diplomas_pending = Diploma.pending`